src/action/notification.js

```diff
@@ -9,15 +9,21 @@ class NotificationAction {
 
   display({ type, msg, wait, err, handler, handlerLbl }) {
     if (err) log.error(msg, err);
-    this._store.notifications.push({
-      type: type || (err ? 'error' : 'info'),
-      message: msg,
-      waiting: wait,
-      date: new Date(),
-      handler: handler || (err ? () => this._nav.goCLI() : null),
-      handlerLbl: handlerLbl || (err ? 'Show error logs' : null),
-      display: true,
-    });
+    const ntfnCount = this._store.notifications.length;
+    let prevNtfn = ntfnCount ? this._store.notifications[ntfnCount - 1] : null;
+    if (prevNtfn && prevNtfn.message === msg) {
+      prevNtfn.date = new Date();
+    } else {
+      this._store.notifications.push({
+        type: type || (err ? 'error' : 'info'),
+        message: msg,
+        waiting: wait,
+        date: new Date(),
+        handler: handler || (err ? () => this._nav.goCLI() : null),
+        handlerLbl: handlerLbl || (err ? 'Show error logs' : null),
+        display: true,
+      });
+    }
```

Comment:
This action should probably not need to be changed. Either we call `notification.display` only once in the `info` action (problem is we want the notification bar to stay visible, so this won't work) or we filter the notification list to render the syncing info only once. Both aren't  really solutions I'm happy with. Thoughts?

Reply:
I think it would make sense to modify `computedNotifications` to filter the notification list such that a new `Syncing to chain...` notification is shown every 5 minutes. I'd also be fine with only showing it once. what do you think?

Reply:
Showing it once is fine. Filtering in computedNotifications makes sense. Something like...
```
const all = [];
notifications.forEach(n => {
  if (n.waiting && all.find(a => a.waiting)) {
    return;
  }
  all.push(n);
});
```

Reply:
We only use the `waiting` flag for syncing so that should do the job.